global/post-tasks.d/015cosmos-trust

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
-gnupg_show_options='--import --import-options show-only,import-minimal'
-if [[ $(lsb_release -sr | awk -F . '{ print $1 }') -le 16 ]]; then
+gnupg_show_options=("--import" "--import-options" "show-only,import-minimal")
+if [[ $(lsb_release -si) = "Ubuntu" ]] && [[ $(lsb_release -sr | awk -F . '{ print $1 }') -le 16 ]]; then
     # gpg on Ubuntu 16 and less is gnupg < 2, which doesn't have --import-options show-only
     # but on the other hand defaults to this mode (https://dev.gnupg.org/T2943)
-    gnupg_show_options='--dry-run'
+    gnupg_show_options=("--dry-run")
```

Comment:
Should this be:
```
    gnupg_show_options=("--dry-run")
```

As in over write the array with ("--dry-run") or should it be:
```
    gnupg_show_options+=("--dry-run")
```
As in append "--dry-run" to the array?

Reply:
Absolutely, just wanted to make sure that that was the intent, and your reasoning seems to confirm that it is correct.